<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trim</class>
 <widget class="QWidget" name="trim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trim</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="3">
    <widget class="QWidget" name="side_panel" native="true">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TRIM settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="settings">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="sample_name_label">
           <property name="text">
            <string>Sample Name</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="min_momentum_linedit"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="sim_type_combo">
           <item>
            <property name="text">
             <string>Mono</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Momentum Spread</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="scan_momentum_combo">
           <item>
            <property name="text">
             <string>No</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yes</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="max_momentum_label">
           <property name="text">
            <string>Max Momentum (MeV/c)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="sim_type_label">
           <property name="text">
            <string>Simulation type</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="max_momentum_linedit"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="momentum_spread_linedit"/>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="srim_exe_dir_linedit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="momentum_linedit"/>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="stats_linedit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="sample_name_linedit"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="trim_out_dir_label">
           <property name="text">
            <string>SRIM output directory</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="momentum_label">
           <property name="text">
            <string>Momentum (MeV/c)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="momentum_step_linedit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="momentum_spread_label">
           <property name="text">
            <string>Momentum Spread (%)</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="srim_exe_dir_label">
           <property name="text">
            <string>SRIM.exe directory</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="stats_label">
           <property name="text">
            <string>Stats for optimal run</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="trim_out_dir_linedit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="scan_momentum_label">
           <property name="text">
            <string>Scan Momentum?</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="momentum_step_label">
           <property name="text">
            <string>Momentum Step (MeV/c)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="min_momentum_label">
           <property name="text">
            <string>Min Momentum (MeV/c)</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Layers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="layers_tab">
         <attribute name="title">
          <string>Layers</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="BaseTable" name="layer_setup_table">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <column>
             <property name="text">
              <string>Sample</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Thickness (mm)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Density (g/cm^3)</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="results_tab">
         <attribute name="title">
          <string>Results</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <widget class="BaseTable" name="results_table">
            <column>
             <property name="text">
              <string>Momentum (MeV/c)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Options</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="save_all_sims_button">
            <property name="text">
             <string>Save all</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Implantation</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTreeWidget" name="results_tree">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <column>
             <property name="text">
              <string>Momentum (MeV/c)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Layers</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>% of muons in layer</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string># of muons in layer</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="run_sim_button">
           <property name="text">
            <string>Run Simulations</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QWidget" name="results_panel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="results_label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Results</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="results_tabs">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseTable</class>
   <extends>QTableWidget</extends>
   <header>EVA.widgets.base.base_table</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
